examples/mnist/train_mnist.py

```diff
@@ -85,12 +85,14 @@ def main():
     trainer.extend(extensions.LogReport())
 
     # Save two plot images to the result dir
-    trainer.extend(
-        extensions.PlotReport(['main/loss', 'validation/main/loss'], 'epoch',
-                              file_name='loss.png'))
-    trainer.extend(
-        extensions.PlotReport(['main/accuracy', 'validation/main/accuracy'],
-                              'epoch', file_name='accuracy.png'))
+    if extensions.PlotReport.available:
```

Comment:
Oops, sorry it's a mistake. Another option is to make `available` a property of PlotReport. I'll make a modification based on this.